pkgs/by-name/lo/local-ai/package.nix

```diff
@@ -28,11 +29,11 @@
   # https://github.com/NixOS/rfcs/pull/169
 
   # CPU extensions
-  enable_avx ? true,
-  enable_avx2 ? true,
+  enable_avx ? stdenv.hostPlatform.isx86_64,
```

Comment:
Why not `stdenv.hostPlatform.avxSupport`?

Reply:
Ah, I remember my last PR about this: https://github.com/NixOS/nixpkgs/pull/377424#discussion_r1948125116

Still I think it would be good to make this more explicit.